docs/pt-br/actions.md

```diff
@@ -23,10 +23,10 @@ const store = new Vuex.Store({
 })
 ```
 
-Os manipuladores de ação recebem um objeto de contexto que expõe o mesmo conjunto de métodos / propriedades na instância da _store_ , para que você possa chamar `context.commit` para confirmar uma mutação ou acessar o estado e os getters através do `context.state` e do `contexto. getters`.
-Veremos por que esse objeto de contexto não é a própria instância da _store_  quando apresentamos [Módulos](modules.md) mais tarde.
+Os manipuladores de ação recebem um objeto de contexto que expõe o mesmo conjunto de métodos / propriedades na instância do _store_ , para que você possa chamar `context.commit` para confirmar uma mutação ou acessar o estado e os _getters_  através do `context.state` e do `contexto.getters`.
```

Comment:
Este parágrafo tem um exemplo de dois espaços após a palavra _getters_ e não há pontuação após.

Reply:
Tirei print com e sem espaço. Me diz o que acha melhor que eu aplico pra todos =)
![teste](https://user-images.githubusercontent.com/6675426/39217223-5ee2949e-47f5-11e8-82c4-bdc730607fa8.jpg)

Reply:
@ErickPetru Espaços extras removidos. Faltou os exemplos de itálicos desnecessários que tinha te pedido. Obrigado